CFC_DataCollector/recommender/improve_trip_pipeline.py

```diff
@@ -0,0 +1,28 @@
+######
+#1  Figure out best alternative for each trips
+#1.5 For each trip do:
+#3. Perturbe trips
+#4. Augment perturbed trips
+#5. Apply model to alternate and perturbed trips
+#6. Return list of improved trips
+
+#construction of model' can part of utility model pipeline or this pipeline
+
+#getAugmentedTrips: retrieve augmented trips from database, 
+		#determine perturbed trips,
+		#combine augmented and perturbed trips (which may or may not be cached, tbd)
+######
+def buildRecommendedTrip(user_id, alternative_trips):
+	#get improved trips
+	tripsToImprove = getTripsToImprove(user_uuid, alternate_trips) 
```

Comment:
what module might this be defined in? I am thinking of a trip selection module that has functions to select both types of trips (training set trips and test set trips). Makes sense?